gradle/libs.versions.toml

```diff
@@ -6,7 +6,7 @@ junit = "5.10.1"
 maven-publish = "0.27.0"
 # When updating, make sure to update buf.gen.yaml and src/test/resources/proto/buf.gen.*.yaml versions to match
 # and regenerate code with 'make generate'.
-protobuf = "3.25.1"
+protobuf = "3.25.2"
```

Comment:
Manually ran `make generate` and checked in changes (we have to do this when we update proto).